workshop/content/docs/publishing/ogcapi-maps.md

```diff
@@ -13,6 +13,22 @@ any geospatial data as a georeferenced map image.
 
 pygeoapi supports the OGC API - Maps draft specification, using [MapServer MapScript](https://www.mapserver.org/mapscript) and a WMS facade as core backends.
 
+To expose data using `MapScript` following configuration can be added 
```

Comment:
This section should get people ready to publish data using OGC API - Maps. The current content is a bit minimal. I would expect something similar to [this](https://dive.pygeoapi.io/publishing/ogcapi-features/), or [this](https://dive.pygeoapi.io/publishing/ogcapi-edr/), called "Publishing Raster Data in pygeoapi". Like in those examples, we should explain what we are going to do, which dataset we will use (e.g.: what is it) and where students can find it. Any information about how to run the example should be covered here.

Reply:
What are your thoughts on 

In this section we'll be exposing a shapefile available at `/data/airport/airport.shp` location. This data can be consumed with various clients which can access OGC APIs - Maps. Here we can also pass style in *.sld* format. Which can be generated on [Geoserver](https://docs.geoserver.org/stable/en/user/styling/index.html), [QGIS](https://www.qgistutorials.com/en/docs/3/basic_vector_styling.html), etc.